### PR TITLE
EZP-25192: add field definition identifier to the dom for field views and field edit views

### PR DESCRIPTION
**Pull request by mhyndle**
> Jira: https://jira.ez.no/browse/EZP-25192
> status: ready for review

## Description
To ease a bit BDD tests, this PR adds `data-field-definition-identifier` attribute to field views and field edit views.

## Tests
manual + unit tests

### CODE REVIEW

**Comment by mhyndle** [comment]
ready for review
ping @dpobel @yannickroger @StephaneDiot

**Comment by andrerom** [comment]
+1 /cc @pcardiga @miguelcleverti

**Comment by dpobel** [comment]
@andrerom also cc @bdunogier, he asked for that.

File: Resources/public/js/views/ez-fieldeditview.js [hunk 1]
```diff
@@ -67,8 +67,9 @@ YUI.add('ez-fieldeditview', function (Y) {
          * @return {eZ.FieldEditView}
          */
         render: function () {
-            var defaultVariables = {
-                    fieldDefinition: this.get('fieldDefinition'),
+            var def = this.get('fieldDefinition'),
+                defaultVariables = {
```
Comment (dpobel) [inline Resources/public/js/views/ez-fieldeditview.js]: the issue was only about field views but I guess it's OK to do that on edit views as well.

**Comment by dpobel** [comment]
+1

**Comment by miguelcleverti** [comment]
+1

**Pull request merged**